docs/DEVELOPING.md

```diff
@@ -172,9 +167,7 @@ The high level flow to upgrade to a newer version of Polkadot is:
     ```shell
     export POLKADOT_NEW_VERSION="<version>"; # for example, 0.9.32
 
-    diener update --polkadot --branch release-v$POLKADOT_NEW_VERSION;
-    diener update --substrate --branch polkadot-v$POLKADOT_NEW_VERSION;
-    diener update --cumulus --branch polkadot-v$POLKADOT_NEW_VERSION;
+    diener update --polkadot-sdk --branch release-v$POLKADOT_NEW_VERSION;
```

Comment:
Just notify this behavior has not been merged yet: https://github.com/paritytech/diener/pull/45

Reply:
Are you fine with keeping as proposed and iterating over this as part of the Polkadot SDK updates after v1.1.0?

Reply:
Yeah! Sure, I was only notifying

Reply:
Click the button 🚀